From d8543eca98ffd2e7527836d43a836ec740076dd6 Mon Sep 17 00:00:00 2001
From: Ahmad Abbas <ahmada@marvell.com>
Date: Tue, 12 Jan 2016 17:43:54 +0200
Subject: [PATCH 39/66] config: replaced armv7 be defconfig with armv8 be
 defconfig

armv7 be is not supported by marvell for buildroot 2015.11

Change-Id: I67ffabcb3dceaf1ba13c4405060f33581e32d1a3
Signed-off-by: Ahmad Abbas <ahmada@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/26737
Reviewed-by: Shadi Ammouri <shadi@marvell.com>
Tested-by: Shadi Ammouri <shadi@marvell.com>
---
 configs/mvebu_armv7_be_defconfig | 74 ----------------------------------------
 configs/mvebu_armv8_be_defconfig | 70 +++++++++++++++++++++++++++++++++++++
 2 files changed, 70 insertions(+), 74 deletions(-)
 delete mode 100644 configs/mvebu_armv7_be_defconfig
 create mode 100644 configs/mvebu_armv8_be_defconfig

diff --git a/configs/mvebu_armv7_be_defconfig b/configs/mvebu_armv7_be_defconfig
deleted file mode 100644
index 1d90af1..0000000
--- a/configs/mvebu_armv7_be_defconfig
+++ /dev/null
@@ -1,74 +0,0 @@
-BR2_armeb=y
-BR2_cortex_a9=y
-BR2_ARM_EABIHF=y
-BR2_ARM_FPU_VFPV3=y
-BR2_TOOLCHAIN_EXTERNAL=y
-BR2_TOOLCHAIN_EXTERNAL_CUSTOM=y
-BR2_TOOLCHAIN_EXTERNAL_PATH=""
-BR2_TOOLCHAIN_EXTERNAL_CUSTOM_PREFIX="arm-marvell-linux-gnueabi"
-BR2_TOOLCHAIN_EXTERNAL_GCC_4_6=y
-BR2_TOOLCHAIN_EXTERNAL_HEADERS_3_2=y
-BR2_TOOLCHAIN_EXTERNAL_CUSTOM_GLIBC=y
-BR2_TOOLCHAIN_EXTERNAL_CXX=y
-BR2_MVEBU_RELEASE_NAME="2016_T1.0.eng_drop_v1"
-BR2_TARGET_GENERIC_GETTY_PORT="ttyS0"
-BR2_ROOTFS_OVERLAY="user_files/"
-BR2_PACKAGE_BUSYBOX_SHOW_OTHERS=y
-BR2_PACKAGE_BZIP2=y
-BR2_PACKAGE_GZIP=y
-BR2_PACKAGE_BONNIE=y
-BR2_PACKAGE_DHRYSTONE=y
-BR2_PACKAGE_FIO=y
-BR2_PACKAGE_LMBENCH=y
-BR2_PACKAGE_MEMSTAT=y
-BR2_PACKAGE_STRACE=y
-BR2_PACKAGE_STRESS=y
-BR2_PACKAGE_WHETSTONE=y
-BR2_PACKAGE_GREP=y
-BR2_PACKAGE_BTRFS_PROGS=y
-BR2_PACKAGE_CRAMFS=y
-BR2_PACKAGE_E2FSPROGS_DEBUGFS=y
-BR2_PACKAGE_XFSPROGS=y
-BR2_PACKAGE_DEVMEM2=y
-BR2_PACKAGE_HDPARM=y
-BR2_PACKAGE_I2C_TOOLS=y
-BR2_PACKAGE_IOSTAT=y
-BR2_PACKAGE_LM_SENSORS=y
-BR2_PACKAGE_LSHW=y
-BR2_PACKAGE_MDADM=y
-BR2_PACKAGE_MEMTESTER=y
-BR2_PACKAGE_PARTED=y
-BR2_PACKAGE_SDPARM=y
-BR2_PACKAGE_SMARTMONTOOLS=y
-BR2_PACKAGE_SYSSTAT=y
-BR2_PACKAGE_BC=y
-BR2_PACKAGE_ARPTABLES=y
-BR2_PACKAGE_BRIDGE_UTILS=y
-BR2_PACKAGE_DHCPCD=y
-BR2_PACKAGE_ETHTOOL=y
-BR2_PACKAGE_IFTOP=y
-BR2_PACKAGE_IPERF=y
-BR2_PACKAGE_IPERF3=y
-BR2_PACKAGE_IPSEC_TOOLS=y
-BR2_PACKAGE_IPTABLES=y
-BR2_PACKAGE_LIGHTTPD=y
-BR2_PACKAGE_NCFTP=y
-BR2_PACKAGE_NETCAT=y
-BR2_PACKAGE_RSYNC=y
-BR2_PACKAGE_SAMBA4=y
-BR2_PACKAGE_TCPDUMP=y
-BR2_PACKAGE_VSFTPD=y
-BR2_PACKAGE_WGET=y
-BR2_PACKAGE_WIRESHARK=y
-BR2_PACKAGE_BASH=y
-BR2_PACKAGE_FILE=y
-BR2_PACKAGE_TIME=y
-BR2_PACKAGE_WHICH=y
-BR2_PACKAGE_FTOP=y
-BR2_PACKAGE_HTOP=y
-BR2_PACKAGE_KVMTOOL=y
-BR2_PACKAGE_TAR=y
-BR2_PACKAGE_LESS=y
-BR2_PACKAGE_VIM=y
-BR2_TARGET_ROOTFS_TAR_GZIP=y
-BR2_PACKAGE_HOST_MKE2IMG=y
diff --git a/configs/mvebu_armv8_be_defconfig b/configs/mvebu_armv8_be_defconfig
new file mode 100644
index 0000000..72a3f7f
--- /dev/null
+++ b/configs/mvebu_armv8_be_defconfig
@@ -0,0 +1,70 @@
+BR2_aarch64_be=y
+BR2_TOOLCHAIN_EXTERNAL=y
+BR2_TOOLCHAIN_EXTERNAL_PATH="/path/to/compile"
+BR2_TOOLCHAIN_EXTERNAL_CUSTOM_PREFIX="aarch64_be-marvell-linux-gnu"
+BR2_TOOLCHAIN_EXTERNAL_GCC_5=y
+BR2_TOOLCHAIN_EXTERNAL_HEADERS_3_18=y
+BR2_TOOLCHAIN_EXTERNAL_CUSTOM_GLIBC=y
+BR2_TOOLCHAIN_EXTERNAL_CXX=y
+BR2_MVEBU_RELEASE_NAME="2016_T1.0.eng_drop_v1"
+BR2_TARGET_GENERIC_GETTY_PORT="ttyS0"
+BR2_ROOTFS_OVERLAY="user_files/"
+BR2_PACKAGE_BUSYBOX_SHOW_OTHERS=y
+BR2_PACKAGE_BZIP2=y
+BR2_PACKAGE_GZIP=y
+BR2_PACKAGE_BONNIE=y
+BR2_PACKAGE_DHRYSTONE=y
+BR2_PACKAGE_FIO=y
+BR2_PACKAGE_LMBENCH=y
+BR2_PACKAGE_MEMSTAT=y
+BR2_PACKAGE_STRACE=y
+BR2_PACKAGE_STRESS=y
+BR2_PACKAGE_WHETSTONE=y
+BR2_PACKAGE_GREP=y
+BR2_PACKAGE_BTRFS_PROGS=y
+BR2_PACKAGE_CRAMFS=y
+BR2_PACKAGE_E2FSPROGS_DEBUGFS=y
+BR2_PACKAGE_XFSPROGS=y
+BR2_PACKAGE_DEVMEM2=y
+BR2_PACKAGE_HDPARM=y
+BR2_PACKAGE_I2C_TOOLS=y
+BR2_PACKAGE_IOSTAT=y
+BR2_PACKAGE_LM_SENSORS=y
+BR2_PACKAGE_LSHW=y
+BR2_PACKAGE_MDADM=y
+BR2_PACKAGE_MEMTESTER=y
+BR2_PACKAGE_PARTED=y
+BR2_PACKAGE_SDPARM=y
+BR2_PACKAGE_SMARTMONTOOLS=y
+BR2_PACKAGE_SYSSTAT=y
+BR2_PACKAGE_DTC=y
+BR2_PACKAGE_BC=y
+BR2_PACKAGE_ARPTABLES=y
+BR2_PACKAGE_BRIDGE_UTILS=y
+BR2_PACKAGE_DHCPCD=y
+BR2_PACKAGE_ETHTOOL=y
+BR2_PACKAGE_IFTOP=y
+BR2_PACKAGE_IPERF=y
+BR2_PACKAGE_IPERF3=y
+BR2_PACKAGE_IPSEC_TOOLS=y
+BR2_PACKAGE_IPTABLES=y
+BR2_PACKAGE_LIGHTTPD=y
+BR2_PACKAGE_NCFTP=y
+BR2_PACKAGE_NETCAT=y
+BR2_PACKAGE_RSYNC=y
+BR2_PACKAGE_SAMBA4=y
+BR2_PACKAGE_TCPDUMP=y
+BR2_PACKAGE_VSFTPD=y
+BR2_PACKAGE_WGET=y
+BR2_PACKAGE_WIRESHARK=y
+BR2_PACKAGE_BASH=y
+BR2_PACKAGE_FILE=y
+BR2_PACKAGE_TIME=y
+BR2_PACKAGE_WHICH=y
+BR2_PACKAGE_FTOP=y
+BR2_PACKAGE_HTOP=y
+BR2_PACKAGE_TAR=y
+BR2_PACKAGE_LESS=y
+BR2_PACKAGE_VIM=y
+BR2_TARGET_ROOTFS_TAR_GZIP=y
+BR2_PACKAGE_HOST_MKE2IMG=y
-- 
1.9.1

